libobs/util/platform-cocoa.m

```diff
@@ -371,6 +371,19 @@ uint64_t os_get_sys_free_size(void)
 	return vmstat.free_count * vm_page_size;
 }
 
+uint64_t os_get_sys_total_size(void)
+{
+	vm_statistics_data_t vmstat = {};
+	if (!os_get_sys_memory_usage_internal(&vmstat))
+		return 0;
+
+	natural_t used = (vmstat.wire_count + vmstat.active_count +
+			  vmstat.inactive_count) *
+			 vm_page_size;
+	natural_t free = vmstat.free_count * vm_page_size;
+	return used + free;
```

Comment:
Not sure if this should just use the same thing that we use in the code that logs this to the log file, which uses `sysctl` and just gets this info directly from there. This was just reusing the existing code for used memory but in reverse.